MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1992,14 +2016,18 @@ public MediaStream GetMediaStream(IEnumerable<MediaStream> allStream, int? desir
             var isQsvHevcEncoder = outputVideoCodec.Contains("hevc_qsv", StringComparison.OrdinalIgnoreCase);
             var isNvdecDecoder = videoDecoder.Contains("cuda", StringComparison.OrdinalIgnoreCase);
             var isNvencEncoder = outputVideoCodec.Contains("nvenc", StringComparison.OrdinalIgnoreCase);
-            var isTonemappingSupported = IsTonemappingSupported(state, options);
+            var isOpenclTonemappingSupported = IsOpenclTonemappingSupported(state, options);
             var isVppTonemappingSupported = IsVppTonemappingSupported(state, options);
             var isTonemappingSupportedOnVaapi = string.Equals(options.HardwareAccelerationType, "vaapi", StringComparison.OrdinalIgnoreCase) && isVaapiDecoder && (isVaapiH264Encoder || isVaapiHevcEncoder);
             var isTonemappingSupportedOnQsv = string.Equals(options.HardwareAccelerationType, "qsv", StringComparison.OrdinalIgnoreCase) && isVaapiDecoder && (isQsvH264Encoder || isQsvHevcEncoder);
+            var isCudaFormatConversionSupported = _mediaEncoder.SupportsFilter("scale_cuda", "Output format (default \"same\")");
```

Comment:
I haven't looked at `SupportsFilter` yet, but is the 2nd string necessary?

Reply:
Yes. Only our custom jellyfin-ffmpeg and ffmpeg 4.5 devel have `format` option in scale_cuda. The 2nd string is used to check that field.